components/skus/browser/rs/lib/src/sdk/credentials/mod.rs

```diff
@@ -134,18 +134,19 @@ where
         &self,
         domain: &str,
     ) -> Result<Option<CredentialSummary>, SkusError> {
-        if let Some(orders) = self.client.get_orders().await? {
-            for order in orders {
-                if order.location_matches(&self.environment, domain) {
-                    if let Some(value) = self
-                        .matching_order_credential_summary(&order.id, domain)
-                        .await?
-                    {
-                        return Ok(Some(value));
-                    }
-                }
+      if let Some(orders) = self.client.get_orders().await? {
+        for order in orders {
+          if order.location_matches(&self.environment, domain) {
+            let wrapped_value = self
```

Comment:
Rust noob question here; if there any advantage to the way it was setting this before using `if let`?